README.md

```diff
@@ -1,2 +1,189 @@
+  ·
+  <a href="mailto:security@appsembler.com">Report security issues</a>
+  ·
+  <a href="https://www.appsembler.com/">Appsembler</a>
+  ·
+  <a href="https://www.appsembler.com/blog/">Blog</a>
+</p>
+
+
+## Gestore
+
+A set of tools that will help you to 
+- Export individual objects from DB. 
+- Import exported objects back.
+- Delete objects from the database and all other objects related to it.
+
+## Table of Contents
+1. [Why using this tool](#why-using-this-tool)
+    1. [Gestore vs Django dumpdata and loaddata](#gestore-vs-django-dumpdata-and-loaddata)
+1. [Get started](#get-started)
+1. [How does it work](#how-does-it-work)
+    1. [Export functionality](#export-functionality)
+    1. [Import functionality](#import-functionality)
+    1. [Delete functionality](#delete-functionality)
+    1. [Demo app](#demo-app)
+1. [Releasing](#releasing)
+1. [Challenges](#challenges)
+
+> **Note**
+> 
+> Object Import and Delete are not production-ready yet. Use with caution
+
+## Why using this tool
+This idea came out of Appsembler Multi-Cluster UI/UX Workflows. This tool is handy for supporting multiple clusters. 
+Other reasons why having robust Export/Import/Delete functionality on your app would be highly beneficial:
+- Frees your site from lots of data you are not using. It's a great idea to export such data to a file system so you can import it later.
+- Decreases the overhead for data tools.
+- Removes old data to keep your costs down and improve performance of aggregation functions (e.g., data pipelines)
+- Deletes obsolete objects as customers churn.
+- Data export is beneficial for GDPR reasons
+- Some customers want their data now for DR (disaster recovery) reasons, not because they're churning.
+- If you are strongly motivated to create a separate cluster for data that already exists on a current one.
+- Lowers the risk of objects (e.g., trial users) being able to crack your site isolation and access data from paying customers.
+
+### Gestore vs Django dumpdata and loaddata
+While the functionality might seem the same, these Gestore commands are entirely different from Django commands.
+
+You can use Django's `dumpdata` commands to back up (export) your models or whole database. And `loaddata` command helps to import these objects back.
+
+On the other hand, Gestore's `exportobjects` command will help you export all data across the database that's only related to a given Object. This functionality will make sure that you can import these exported objects successfully later.
+
+### Example
+Let's assume you want to export a specific `Books` object:
+
+<img width="965" alt="Screen Shot 2021-06-28 at 11 16 15 AM" src="https://user-images.githubusercontent.com/11036472/123684405-43905a00-d802-11eb-862b-abc0392b4bf6.png">
+
+In Django, you have to export all Books objects `dumpdata` and load it back later using `loaddata`. This method is not practical in two situations:
+- You only want to export one object, not the whole table.
```

Comment:
FWIW, `dumpdata` also supports exporting a single object or list of objects: https://docs.djangoproject.com/en/3.1/ref/django-admin/#cmdoption-dumpdata-pks